pymc3/distributions/continuous.py

```diff
@@ -3471,6 +3454,21 @@ def logcdf(
         )
 
 
+class RiceRV(RandomVariable):
+    name = "rice"
+    ndim_supp = 0
+    ndims_params = [0, 0]
+    dtype = "floatX"
+    _print_name = ("Rice", "\\operatorname{Rice}")
+
+    @classmethod
+    def rng_fn(cls, rng, nu, sigma, size=None):
+        return stats.rice.rvs(b=nu / sigma, scale=sigma, size=size, random_state=rng)
```

Comment:
Since the `logp` needs `b` anyway, maybe it makes sense to make the default parametrization in terms of `b` and `sigma` so that we can at least avoid the division `nu/sigma` in the scipy call. What do you think?

Reply:
Actually it seems we wouldn't even need `nu` in the `logp` altogether.

Reply:
I agree because the scipy function use `b` as the default, but should we adjust the documentation accordingly? Since it assumes that `b` is an alternative to the `nu`, rather than being the default parameter to pass.

Reply:
No need to change the documentation. It doesn't affect the behavior users see